rcaracaus/pledge-vegan-node
# hotfix for incorrect link to contact us page on homepage

Steps to test:

1. Go to homepage localhost:3000
2. Click on contact us
3. Check that url links to /contact-us and page loads correctly
